package.json

```diff
@@ -15,27 +15,41 @@
   },
   "dependencies": {
     "ajv": "^6.4.0",
-    "chalk": "^2.4.0",
+    "body-parser": "^1.18.2",
+    "chalk": "^2.3.2",
     "commander": "^2.15.1",
+    "express": "^4.16.3",
     "fs-extra": "^5.0.0",
     "listr": "^0.14.0",
     "lodash": "^4.17.5",
     "pjson": "^1.0.9",
+    "request": "^2.85.0",
+    "request-promise": "^4.2.2",
+    "rimraf": "^2.6.2",
```

Comment:
no need for rimraf (and types/rimraf) here - add rimraf to devdependencies instead.